packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -1,33 +1,49 @@
 <template>
     <nav
         v-if="hasNavigationLinks"
-        class="c-nav c-nav--global"
+        :class="[
+            $style['c-nav'],
+            $style['c-nav--global']
+        ]"
         data-test-id="nav-container">
         <button
-            :class="['c-nav-trigger c-nav-toggle is-hidden--noJS', navToggleThemeClass, {
-                'is-open': navIsOpen
-            }]"
+            :class="[
+                $style['c-nav-trigger'],
+                $style['c-nav-toggle'],
+                $style['is-hidden--noJS'],
+                navToggleThemeClass,
+                {
+                    [$style['is-open']]: navIsOpen
```

Comment:
Just to confirm, the `is-open` classes still work as expected? I see they are chained to other selectors in the styles below and I haven't actually seen this done in CSS modules before.

Reply:
👀  I'll triple check, thanks